README.md

```diff
@@ -39,7 +39,26 @@ We use images based on a Kolla build, but we've dropped their Ansible approach t
 favour of Helm and native Kubernetes specs. We also have a number of vendor and self developed extensions, especially
 for Neutron, which are build into our images (you can see reference to many of these in the configuration files/values).
 
+## Installation
 
+### Prerequisites
+
+**Kubernetes with Helm**
+
+See https://github.com/kubernetes/kubernetes and https://github.com/kubernetes/helm
+
+**Local Charts server**
+
+During installation, openstack-helm expects a chart server running on localhost. You can start one with the following command:
+
+    helm serve --address localhost:8879
```

Comment:
I don't get the difference between 127.0.0.1 and localhost in this case, it should be the same. I just to `helm serve` and using the makefile in this repo works for me. What error are you getting when you run it without the `--address localhost:8879`